packages/core/src/axis-chart.ts

```diff
@@ -74,7 +77,8 @@ export class AxisChart extends Chart {
 			}
 		};
 
-		if (zoomBarEnabled) {
+		// if all zoom bars are locked, no need to add chart brush
+		if (zoomBarEnabled && !isZoomBarLocked) {
```

Comment:
don't think we really need a `locked` flag, this can simply be `if (zoomBarEnabled && !isZoomBarLoading) {`

Reply:
Please refer to https://github.com/carbon-design-system/carbon-charts/pull/776#discussion_r491806328, `locked` state is different from `loading` state.